### PR TITLE
Fixed  #5231

### LINKED ISSUE #5231: User name for GCVote can not be deleted
2015.10.08-NB:
- Open Settings/Services
- Open GCVote settings
- Click on username and replace the existing user by an empty field
- Select OK
- Select "Verify login"

This will return "Check OK" and I do see in the log that the previously stored username is still in use
- Close c:geo / Force close
- Start c:geo again
- Go to Settings/Services/GCVote

You will see, that the previous username has been restored.

As the username of GCVote and geocaching.com need to be identical we most probably derive it from the settings for geocaching.com.
However couldn't we just disallow editing of the username in the GCVote settings and just show it for informational purposes.

### PR DESCRIPTION
**Pull request by GodfatherThe**
Locked username on GCVote page, added refreshing the preference when changing from GC.com screen.

### CODE REVIEW

**Comment by GodfatherThe** [comment]
@Bananeweizen @rsudev @Lineflyer could you please take a look?

File: main/res/xml/preferences.xml [hunk 1]
```diff
@@ -313,6 +313,7 @@
                         android:text="@string/init_gcvote_password_description" />
 
                     <EditTextPreference
+                        android:enabled="false"
```
Comment (rsudev) [inline main/res/xml/preferences.xml]: Did you consider switching it to a simple (and therefore display-only) Preference? I find the disabled look unfitting.
We likely should also adapt the string to tell that this is in fact the geocaching.com username.

**Comment by GodfatherThe** [comment]
@rsudev If it's usual Preference, then it looks like user is able to change it, but in fact he\she can not, so it can be confusing. However, when it's disabled, it underlines that you're unable to edit this. 
Agree about the string, added. And also I need only login handling, so added a condition.

**Comment by SammysHP** [comment]
rsudev did not mean a normal preference, but a text preference like the one we use for descriptions etc.

**Comment by Bananeweizen** [comment]
Can one of the admins verify this patch?

**Comment by SammysHP** [comment]
The patch might work, but we are still discussing about the method that is used to show that it is disabled.

**Comment by SammysHP** [comment]
@GodfatherThe Are you still interested in this patch?

**Comment by rsudev** [comment]
ok to test

**Comment by samueltardieu** [comment]
No news from submitter and not mergeable as-is, closing, reopen one if you want to start working on this again.

**Pull request closed without merging**